### DISCUSSION TITLE
Code formatter used for mkdocs-material codebase

### DISCUSSION THREAD

**Discussion by jbms** [squidfunk/mkdocs-material#4076] [General]
I'm wondering what code formatters, if any, are used on the mkdocs-material codebase (JavaScript, CSS) so that my own changes can match the existing formatting.

I have seen that `npm run check:style:ts -- --fix` and `npm run check:style:scss -- --fix` do some amount of auto-formatting, but especially the typescript lint operation does not seem to be comprehensive (doesn't fix indentation or break lines).

**Comment by squidfunk** [comment]
Formatting is done manually. I'm not a fan of auto-formatters.
